korman/ui/modifiers/anim.py

```diff
@@ -43,6 +43,9 @@ def animation(modifier, layout, context):
         col.enabled = modifier.loop
```

Comment:
```suggestion
        col.enabled = modifier.loop and modifier.obj_sdl_anim
```

Reply:
This caused everything below "Start Marker" to disappear if Loop Anim was selected. I'll play around with this some more. 😉 

EDIT: putting "and **not**" before modifier.obj_sdl_anim fixed it.